.github/PropertiesTest.java

```diff
@@ -0,0 +1,7 @@
+public class PropertiesTest {
+    public static void main(String[] args)
+        throws Exception {
+        String value = System.getProperty("sun.boot.class.path");
+        System.out.println(value);
+    }
+}
```

Comment:
It had honestly never occurred to me to just do this to find the classpath. Brilliant!

Reply:
Found this randomly buried inside of a stackoverflow answer, actually. Standing on the shoulders of giants and all that :)